.travis.yml

```diff
@@ -101,11 +101,6 @@ jobs:
       before_install: skip
       script: PKG_OS="darwin" make packages-sdk
       deploy: *deploy_anchor
-    - name: "push image to Docker Hub"
-      stage: release
-      script:
-        - PKG_OS=linux make build
-        - DOCKER_PUSH_LATEST=true make docker-push
```

Comment:
Is `docker_image` in Drone pre-configured somewhere to run a `make docker-push`?

Reply:
the push to docker hub from drone is not done with our repo `make`, we use a drone plugin.
- https://github.com/src-d/lookout/pull/162/files#diff-3216dfff0ed3e301453e6799e8c367e2R16
- http://plugins.drone.io/drone-plugins/drone-docker/